apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -190,11 +190,13 @@ error.fail=Authentication Failed!
 error.totp.not.enabled=Enable the TOTP in your Profile. Cannot proceed further without TOTP authentication.
 error.retry=Authentication Failed! Please Retry
 auth.totp=Authenticating with TOTP
-enter.verification.code.got.by.device=Enter the verification code generated by your Mobile Application or got via email.
+enter.verification.code.got.by.device=Enter the verification code generated by your authenticator app
+or.got.via.email=or the one you got via email
```

Comment:
Wouldn't it be better to have two separate complete messages to keep things simple? :)

Reply:
Resolved by 15c2958